src/components/cart.js

```diff
@@ -96,22 +115,56 @@ export default class Cart extends Component {
       classes: this.classes,
       lineItemsHtml: this.lineItemsHtml,
       isEmpty: this.isEmpty,
-      formattedTotal: this.formattedLineItemsSubtotal,
+      formattedTotal: this.formattedTotal,
+      discounts: this.discounts,
       contents: this.options.contents,
       cartNote: this.cartNote,
     });
   }
 
+  get styles() {
+    return this.options.manifest.filter((component) => this.config[component].styles).reduce((hash, component) => {
+      const styles = this.config[component].styles;
+      if (component === 'cart') {
+        if (styles.subtotal) {
+          styles.discountAmount = styles.subtotal;
+        }
+
+        if (styles.subtotalText) {
+          styles.discountText = styles.subtotalText;
+        }
+      }
+      hash[component] = styles;
+      return hash;
+    }, {});
+  }
+
   /**
    * get formatted cart subtotal based on moneyFormat
    * @return {String}
    */
   get formattedTotal() {
-    return formatMoney(this.model.subtotalPrice, this.moneyFormat);
+    const total = this.options.contents.discounts ? this.model.subtotalPriceV2.amount : this.model.lineItemsSubtotalPrice.amount;
+    return formatMoney(total, this.moneyFormat);
   }
 
-  get formattedLineItemsSubtotal() {
-    return formatMoney(this.model.lineItemsSubtotalPrice.amount, this.moneyFormat);
+  get discounts() {
```

Comment:
Is this the cart discount? If so, maybe the discount variable names could be more explicit since it can get confusing between the different sets of discounts.